### DISCUSSION TITLE
How to stop an ongoing device discovery ?

### DISCUSSION THREAD

**Discussion by pythops** [bluez/bluer#127] [Q&A] [answered]
I am looking for a way to stop an ongoing device discovery ?

**Comment by surban** [comment, marked as answer]
The device discovery is stopped when you drop the event stream returned by [`Adapter::discover_devices`](https://docs.rs/bluer/latest/bluer/struct.Adapter.html#method.discover_devices).

**Comment by pythops** [comment]
Thanks
